src/pages/content-edit/head.js

```diff
@@ -1,439 +1,124 @@
+            // Controls
+            m("div", { class : css.actions }, [
+                m("a", {
+                        // Attrs
+                        class  : css.back,
+                        title  : "Back to Listing",
+                        href   : prefix("/listing/" + state.schema.key),
+                        config : m.route
+                    },
+                    m("svg", { class : css.icon },
+                        m("use", { href : icons + "#arrow" })
+                    ),
+                    "Back"
                 ),
-                ctrl.saving ? "SAVING..." : "Save"
-            )
-        ]);
-    }
-
 
-    function mScheduleButton() {
-        return m("button", {
-                // Attrs
-                class : css.schedule,
-                title : "Schedule a publish",
+                m("button", {
+                        // Attrs
+                        class    : css.save,
+                        title    : "Save your changes",
+                        disabled : locked || !state.meta.dirty || null,
 
-                // Events
-                onclick : ctrl.toggle.bind(null, undefined)
-            },
-            m("svg", { class : css.onlyIcon },
-                m("use", { href : icons + "#schedule" })
-            )
-        );
-    }
-
-    function mPublishButton() {
-        var isDisabled = false;
-
-        // TODO Better implementation.
-        // if(ctrl.start.ts && isPast(ctrl.start.ts)) {
-        //     isDisabled = true;
-        // }
+                        // Events
+                        onclick : state.ui.saving ?
+                            null :
+                            function() {
+                                content.save();
+                            }   
+                    },
+                    m("svg", { class : css.icon },
+                        m("use", { href : icons + "#save" })
+                    ),
+                    state.ui.saving ? "SAVING..." : "Save"
+                )
+            ]),
 
-        return m("div", { class : css.publishContainer },
-            m("button", {
-                    // Attrs
-                    class    : css.publish,
-                    title    : future ? "Schedule publish" : "Already published",
-                    disabled : locked || isDisabled || null,
+            // Publishing
+            m("div", { class : css.publishing },
+                // Schedule
+                m("button", {
+                        // Attrs
+                        class : css.schedule,
+                        title : "Schedule a publish",
 
-                    // Events
-                    onclick : ctrl.publish.bind(null, options)
-                },
-                m("svg", { class : css.icon },
-                    m("use", { href : icons + (future ? "#schedule" : "#publish") })
+                        // Events
+                        onclick : function() {
```

Comment:
I guess not. Sorry about that. I would have sworn I was having some problems with `context` associated with using these Mithril `on...` handlers, and that wrapping changed it, but it seems like everything is working fine without it. *shrug* My bad?